### ISSUE TITLE
SpriteAnimation Aseprite support not working

### ISSUE THREAD

**Issue by erickzanardo** [flame-engine/flame#566]
## Description

Creating a SpriteAnimation from aseprite data is not working, for some reason, this line is breaking the `SpriteAnimation.fromAsepriteData` constructor.

```dart
    final jsonFrames = jsonData['frames'] as Map<String, Map<String, dynamic>>;
```
## Development environment

Flame version: v1.0.0-rc3 (not published yet)

Flutter doctor:

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel beta, 1.24.0-10.2.pre, on macOS 11.0.1 20B29 darwin-x64, locale pt-BR)
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
[✓] Xcode - develop for iOS and macOS (Xcode 12.1)
[✓] Chrome - develop for the web
[✓] Android Studio (version 4.0)
[✓] IntelliJ IDEA Community Edition (version 2020.2.1)
[✓] Connected device (4 available)```

## Runtime

This issue is related to running on which platform? (Select all that apply)

 - [x] Android :android:
 - [x] iOS :apple:
 - [x] Web :browser:

## Minimal reproducible code (Required for bugs)

Flame aseprite example on Flame repository

**Comment by spydon** [comment]
Is this working in `v1.0.0-rc2`?

**Comment by erickzanardo** [comment]
> Is this working in `v1.0.0-rc2`?

To be honest, I haven't tested, but the line which is breaking is the same since always, I think this is a Flutter change, because it breaks on a casting statement.

**Comment by erickzanardo** [comment]
Fixed on 1.0.0-rc6
